tests/library/global-fetch-cookie.spec.ts

```diff
@@ -143,10 +143,10 @@ it('should send secure cookie over http for localhost', async ({ request, server
     res.setHeader('Set-Cookie', ['a=v; secure', 'b=v']);
     res.end();
   });
-  await request.get(`${server.PREFIX}/setcookie.html`);
+  await request.get(`${server.CROSS_PROCESS_PREFIX}/setcookie.html`);
   const [serverRequest] = await Promise.all([
     server.waitForRequest('/empty.html'),
-    request.get(server.EMPTY_PAGE)
+    request.get(server.CROSS_PROCESS_PREFIX + '/empty.html')
```

Comment:
This should fail I believe as we only check for 'localhost' in [the code](https://github.com/microsoft/playwright/blob/8c4fc62e8f601c81c16fc382f37999dfd4ce1213/packages/playwright-core/src/server/cookieStore.ts#L31).